packages/lesswrong/components/editor/RateLimitWarning.tsx

```diff
@@ -32,15 +53,22 @@ const RateLimitWarning = ({lastRateLimitExpiry, rateLimitMessage}: {
     return `${diffInWeeks} week${diffInWeeks > 1 ? 's' : ''}`
   }
 
-  let message = `Please wait ${getTimeUntilNextPost()} before posting again. ${rateLimitMessage ?? ''}`
+  let message = `<p>You can next post ${getTimeUntilNextPost()} from now.</p> ${rateLimitMessage ?? ''}`
   if (isEAForum) {
     message = `You've written more than 3 comments in the last 30 minutes. Please wait ${getTimeUntilNextPost()} before commenting again. ${rateLimitMessage ?? ''}`
   }
 
-  return <Components.WarningBanner message={message} />
+  if (isEAForum) {
```

Comment:
I think in most cases we (LW team) should forum-gate by conditioning on `isLW`, rather than anything else

Reply:
But I want our forum-gates on Alignment Forum.

Reply:
Oh, I see.  Maybe the correct thing to do in that case is to have something like `isLWorAF` (to avoid forgetting to do `isLW || isAF` each time).

Making it explicit this way seems better because:
1. it makes it obvious which forum the feature "belongs" to
2. it doesn't accidentally opt other forums using the codebase in to the feature by default (though this is a minor consideration)

Reply:
In this case my actual belief is that this should be the default display for rate limits. (though we don't currently have great process for deciding how third-party-forums display stuff that LW and EAforum disagree on)

(I chatted with Robert and one of his wishes was, if this _is_ the default way of displaying things, it be listed first. But that requires the if/then to be if (!EAForum) which feels a bit weird)